src/components/JosekiAdmin/JosekiAdmin.tsx

```diff
@@ -30,7 +30,7 @@ import { JosekiStatsModal } from "JosekiStatsModal";
 
 
 interface JosekiAdminProps {
-    godojo_headers: any;
+    oje_headers: any;
```

Comment:
This can be typed as [`HeadersInit`](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.requestinit.html#headers)

Reply:
Done throughout.

Reply:
Rock on brother!